server.js

```diff
@@ -1,30 +1,92 @@
 import express from "express";
 import cors from "cors";
+import booksData from "./data/books.json";
 
-// If you're using one of our datasets, uncomment the appropriate import below
-// to get started!
-// import avocadoSalesData from "./data/avocado-sales.json";
-// import booksData from "./data/books.json";
-// import goldenGlobesData from "./data/golden-globes.json";
-// import netflixData from "./data/netflix-titles.json";
-// import topMusicData from "./data/top-music.json";
-
-// Defines the port the app will run on. Defaults to 8080, but can be overridden
-// when starting the server. Example command to overwrite PORT env variable value:
-// PORT=9000 npm start
 const port = process.env.PORT || 8080;
 const app = express();
 
-// Add middlewares to enable cors and json body parsing
 app.use(cors());
 app.use(express.json());
 
-// Start defining your routes here
+// Documentación de la API
 app.get("/", (req, res) => {
-  res.send("Hello Technigo!");
+  res.json({
+    message: "Welcome to the Books API",
+    routes: [
+      { method: "GET", endpoint: "/books", description: "Get all books" },
+      { method: "GET", endpoint: "/books/:id", description: "Get a single book by ID" },
+      { method: "GET", endpoint: "/books/search", description: "Search books by query parameters" },
+      { method: "GET", endpoint: "/books/page", description: "Get paginated books" },
+    ],
+  });
 });
```

Comment:
Next week you might want to look into Express List Endpoints (see instructions) so that you don't have to update your docs manually when you change something